Packet++/header/NflogLayer.h

```diff
@@ -0,0 +1,264 @@
+#ifndef PACKETPP_NFLOG_LAYER
+#define PACKETPP_NFLOG_LAYER
+
+#include "Layer.h"
+#include "TLVData.h"
+#include "GeneralUtils.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+
+	/** IPv4 protocol */
+	#define PCPP_WS_NFPROTO_IPV4		2
+	/** IPv6 protocol */
+	#define PCPP_WS_NFPROTO_IPV6		10
+
+	/**
+	 * @struct nflog_header
+	 * Represents Nflog header
+	 */
+#pragma pack(push, 1)
+	struct nflog_header
+	{
+		uint8_t addressFamily;
+		uint8_t version;
+		uint16_t resourceId;
+	};
+#pragma pack(pop)
+
+	/**
+	 * @struct nflog_tlv
+	 * Represents Nflog tlv structure
+	 */
+#pragma pack(push, 1)
+	struct nflog_tlv
+	{
+		/** tlv length */
+		uint16_t tlvLength;
+
+		/** tlv type */
+		uint16_t tlvType;
+	};
+#pragma pack(pop)
```

Comment:
We don't really use this struct. The only place we use it is here: https://github.com/jafar75/PcapPlusPlus/blob/4e06ec9d58a27706c85af4e82aad0324eb418885/Packet%2B%2B/src/NflogLayer.cpp#L53

I think we can remove it from this file and simply do:

```cpp
size_t payloadLen = payloadInfo.getTotalSize() - sizeof(uint16_t) * 2;
```